src/Sylius/Bundle/VariationBundle/Form/Type/OptionValueType.php

```diff
@@ -43,11 +43,11 @@ public function __construct($dataClass, array $validationGroups, $variableName)
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('value', 'text', [
-                'label' => 'sylius.form.option_value.value',
+            ->add('translations', 'a2lix_translationsForms', [
+                'form_type' => sprintf('sylius_%s_option_value_translation', $this->variableName),
+                'label' => 'sylius.form.option.presentation',
             ])
-            ->addEventSubscriber(new AddCodeFormSubscriber())
-        ;
+            ->addEventSubscriber(new AddCodeFormSubscriber());
```

Comment:
I am pretty sure we have convention to leave semicolon in a new line aligned to builder variable call.